2-Draft-Accepted/RFC00NN-PowerShellCore-Installer-Module.md

```diff
@@ -0,0 +1,78 @@
+---
+RFC: RFC00NN
+Author: Joey Aiello
+Status: Draft-Accepted
+SupercededBy: N/A
+Version: 0.1
+Area: External Module
+Comments Due: 7/15/2017
+---
+
+# PowerShellCore Installer/Utility Module
+
+A module called `PowerShellCore` that ships on the Gallery and eases installation and configuration of PowerShell Core on Windows.
+
+## Motivation
+
+Installation and configuration of PowerShell Core on Windows is tricky due to the wide range of Windows versions that we support (7-10 and Server 2008R2-2016) and because of the usage of some "system" concepts shared by Windows PowerShell like the WinRM remoting plugin.
```

Comment:
Not sure why you have double quotes around "system", and concepts should be `components`